homeassistant/components/huawei_lte/__init__.py

```diff
@@ -103,6 +105,8 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+SERVICE_SCHEMA = vol.Schema({vol.Required(CONF_URL): cv.url})
```

Comment:
Should we allow users to not pass a url if they have only a single instance ?

Reply:
Sure, why not. Not sure if there's a clean way to accomplish this via service data schemas, so I opted to make it optional on that level and log error messages when not applicable instead.